config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml

```diff
@@ -0,0 +1,43 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes/kubernetes:
+    # The name should be changed to match the repo name above
+    - name: post-kubernetes-push-images
+      cluster: test-infra-trusted
```

Comment:
Can we add a TODO to move this to a more appropriate cluster?

Reply:
Done